GaugeKit.xcodeproj/project.pbxproj

```diff
@@ -413,6 +416,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "SW.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
ohm, one more thing,
"By default, if you have a pre-Swift 4 project, you will get warnings about this. In a Swift 4 project, you will get build errors. This is controlled via the `SWIFT_SWIFT3_OBJC_INFERENCE` build setting. In a pre-Swift 4 project this is set to `On`. **I would recommend to set this to `Default` (or `Off`), which is now the default option on a new project.** "

Reply:
https://stackoverflow.com/questions/44379348/the-use-of-swift-3-objc-inference-in-swift-4-mode-is-deprecated

Reply:
of fixed